src/Microsoft.Diagnostics.Monitoring/Contracts/IDiagnosticServices.cs

```diff
@@ -17,7 +17,16 @@ public interface IDiagnosticServices : IDisposable
         Task<OperationResult<Stream>> GetDump(int? pid, DumpType mode);
 
         //TODO We can most likely unify trace, cpu, and logs/metrics around one call with the appropriate config
-        Task<OperationResult<Stream>> StartCpuTrace(int? pid, int duration, CancellationToken token);
+        Task<OperationResult<IStreamResult>> StartCpuTrace(int? pid, int duration, CancellationToken token);
+
```

Comment:
@noahfalk Per our discussion this facade would go away at some point, but I'm keeping it for now.